Traceback (most recent call last):
  File "main.py", line 68, in <module>
    loss = Loss(loss_name)
NameError: name 'Loss' is not defined